ERCS/erc-6229.md

```diff
@@ -0,0 +1,321 @@
+---
+eip: 6229
+title: Tokenized Vaults with Lock-in Period
+description: ERC-4626 Tokenized Vaults with Lock-in Period.
```

Comment:
Your description doesn't add much new information that wasn't already in the title. Try to elaborate on the idea you introduce there.

Reply:
I see. I'll try to elaborate more information in description.